website/docs/reference/integrations/datadog.md

```diff
@@ -28,19 +27,40 @@ You can choose to trigger updates for the following events:
 - feature-stale-off
 - feature-environment-enabled
 - feature-environment-disabled
+- feature-variants-updated
+- feature-potentially-stale-on
 
-> *) Deprecated, and will not be used after transition to environments in Unleash v4.3
-
-#### Parameters {#parameters}
+### Parameters {#parameters}
```

Comment:
I wonder about this change - Was it on purpose?

Reply:
Yeah it was to differentiate the header from the emboldened parameter headers - they looked the same. Might be improved if I put the original bullet point lists back in